scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -929,7 +932,8 @@ Function RunTempSensorTest
         -p `"$ContainerRegistryPassword`" ``
         -t `"${ArtifactImageBuildNumber}-windows-$(GetImageArchitectureLabel)`" ``
         -tw `"$TwinTestFileArtifactFilePath`" ``
-        --optimize_for_performance=`"$OptimizeForPerformance`""
+        --optimize_for_performance=`"$OptimizeForPerformance`" ``
+        --bypass_edge_installation=`"$BypassEdgeInstallation`""
```

Comment:
Currently each test will call TestSetup method which will call CleanUp method to uninstall iotedge.  Please update the logic.  btw, when update CleanUp method, please help to move Uninstall-SecurityDaemon that line before docker system prune line.

Reply:
Good point. Updated _TestSetup_ logic and swapped the order of _iotedge_ uninstall and _docker_ prune.